src/cmd/cgo/out.go

```diff
@@ -1669,6 +1669,9 @@ const cBytesDef = `
 // freed, such as by calling C.free (be sure to include stdlib.h
 // if C.free is needed).
 func _Cfunc_CBytes(b []byte) unsafe.Pointer {
+	if len(b) > (1<<30) {
```

Comment:
I'm very confused that how did you find this 1<<30 threshold? Such an accurate value. I also have a panic problem with cgo now. I'm using an old version, I added your judgment to my code, and it still didn't solve my problem. I'm more curious which way did you find this value? @cuiweixie